python-main.js

```diff
@@ -259,6 +262,41 @@ function blocks() {
     return blocklyWrapper;
 }
 
+/*
+ * Please add description here.
```

Comment:
Sorry, forgot to mention I left this one for you to fill.

Reply:
Its all good, added a description.